### PR TITLE
Log out and log in as a different user

### LINKED ISSUE #71: Log out and log in as a different user
Need to be able to logout (deleting the history) and log in as a different user.

### PR DESCRIPTION
**Pull request by santhoshmanikandan**
Added option to logout from the current user account under About page with Remorse timer option. 

Ref: #71 

PS: Doesn't add Multi-Account support yet

### CODE REVIEW

File: qml/pages/OverviewPage.qml [hunk 1]
```diff
@@ -368,6 +384,21 @@ Page {
                 size: BusyIndicatorSize.Large
             }
         }
+
```
Comment (jgibbon) [inline qml/pages/OverviewPage.qml:387]: Perhaps you could use/change the existing loadingLabel (with a different text, obviously) to reduce the amount of qml items?

**Comment by jgibbon** [comment]
Nice work! 
Didn't do a real built/test, just did a quick scroll through the diff, but from that, it seemed pretty good.

File: qml/pages/OverviewPage.qml (hunk 1 above)
Comment (santhoshmanikandan) [reply qml/pages/OverviewPage.qml]: Thanks for the suggestion @jgibbon, I have merged the two Busy Indicators to one now.

**Comment by Wunderfitz** [comment]
Seems to work really well, thank you very much!

**Pull request merged**